client/src/includes/initTagField.ts

```diff
@@ -0,0 +1,29 @@
+import * as $ from 'jquery'
+/** 
+ *This is a function initTagField
+ *@function
+ *@param {number} id - The object id.
+ *@param {string} autocompleteUrl - The 
+ autoComplete Url.
+ *@param {object} options - Other options 
+ passed to initTagField
+ */
+function initTagField(id: number, autocompleteUrl: string, options: object) {
+    const finalOptions = {
+      autocomplete: { source: autocompleteUrl },
+      preprocessTag(val: any) {
+        // Double quote a tag if it contains a space
+        // and if it isn't already quoted.
+        if (val && val[0] !== '"' && val.indexOf(' ') > -1) {
+          return '"' + val + '"';
+        }
+  
+        return val;
+      },
+      ...options,
+    };
+  
+    $('#' + id).tagit(finalOptions);
```

Comment:
Let's use a little less code if we can avoid it. 

Move the element selector to the top of the function and use non-jquery code to selector. 

Then we can return early and not do any logic we don't need. 
```
const tagFieldElement = document.getElementById(I'd);

if (!tagFieldElement) {
  return;
}
```

Reply:
Done